offchain-modules/bridge-lib/src/transfer/to_ckb.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::Result;
+
+use web3::types::{H160, H256};
+use crate::util::eth_util::{ make_transaction, Web3Client};
+
+pub fn approve(from: H160, to: H160, url: String) -> H256{
+    let mut rpc_client = Web3Client::new(url);
+    let approve_tx = make_transaction(from, to);
```

Comment:
We do not need signature here?

Reply:
it just a demo code. I will actually implement it later.